### PR TITLE
fix(component-testing): Layout jumping issue

### PR DESCRIPTION
**Pull request by dmtrKovalenko**
The jumping issue was coming from our layout was actually higher than the window, we didn't see the scrollbar because of internal `react-split-pane` style `overflow:hidden`. 

The layout was higher because of the resizer styles. The styles that we copied from `react-split-pane` were unsafely refactored and we lost an issue. I added a comment about the importance of these styles so we likely won't make hasty refactoring.

**+** I made iframes header same width due to changing styles of resizer. Looks nicer as well 

Screenshot 2021-02-24 at 12 28 37

### CODE REVIEW

File: packages/runner-ct/src/SpecList/SpecList.tsx [hunk 1]
```diff
@@ -55,7 +52,7 @@ export const SpecList: React.FC<SpecsListProps> = observer((props) => {
       />
       <ul
         data-cy="specs-list"
-        className={cs('specs-list', { 'specs-list_text-selection-disabled': props.disableTextSelection })}
+        className="specs-list"
```
Comment (elevatebart) [inline packages/runner-ct/src/SpecList/SpecList.tsx:55]: How do you disable text-selection now?

File: packages/runner-ct/src/lib/ResizableBox.tsx [hunk 2]
```diff
@@ -24,7 +24,9 @@ export const ResizableBox: React.FC<ResizableBoxProps> = ({
   const startClientXRef = React.useRef(width)
   const resizingRef = React.useRef<HTMLDivElement>(null)
 
-  const handleResize = React.useCallback((e) => {
+  const handleResize = React.useCallback((e: MouseEvent) => {
+    e.preventDefault()
```
Comment (elevatebart) [inline packages/runner-ct/src/lib/ResizableBox.tsx:28]: I see, that could be better. 
Did you observe any change in performance?

**Review by elevatebart — APPROVED** [review]
Good job on finding the glitch.
I prefer the CSS things usually but your JS technique for preventing user selection seems to work.

File: packages/runner-ct/src/lib/ResizableBox.tsx (hunk 2 above)
Comment (elevatebart) [reply packages/runner-ct/src/lib/ResizableBox.tsx:28]: If you have not already done so, please make sure this works the same in all 3 supported browser FF, Edge and Chrome

Comment (dmtrKovalenko) [reply packages/runner-ct/src/lib/ResizableBox.tsx:28]: I didn't, but this approach also prevents text selection inside testing iframe

Comment (elevatebart) [reply packages/runner-ct/src/lib/ResizableBox.tsx:28]: After thought: No need to test edge it will use the same engine as chrome.
I will test FF right now and report back

Comment (elevatebart) [reply packages/runner-ct/src/lib/ResizableBox.tsx:28]: https://user-images.githubusercontent.com/5592465/109038897-d58c2880-7691-11eb-8c90-6b00b5161fcb.mov

Dang it !!!

**Review by elevatebart — CHANGES_REQUESTED** [review]
We should probably use user-select:none on the whole window then remove it once we are done dragging?

**Comment by elevatebart** [comment]
What about using the drag event instead? I am sure it would prevent the selection. It is compatible with most browsers.

File: packages/runner-ct/src/lib/ResizableBox.tsx (hunk 2 above)
Comment (elevatebart) [reply packages/runner-ct/src/lib/ResizableBox.tsx:28]: @dmtrKovalenko
What I would do here is remove the user-select changes from here so that we can merge this bug fix and deal with the issue of selecting the iframe in another ticket or PR.

Comment (lmiller1990) [reply packages/runner-ct/src/lib/ResizableBox.tsx:28]: May need `stopPropogation` for FF

**Review by lmiller1990 — DISMISSED** [review]
One trivial comment but generally seems fine

File: packages/runner-ct/src/lib/ResizableBox.tsx (hunk 2 above)
Comment (dmtrKovalenko) [reply packages/runner-ct/src/lib/ResizableBox.tsx]: Fixed this for all the browsers in a much better way. Thanks, folks for this comment!

**Review by elevatebart — APPROVED** [review]
Cool, I would have done it another way but this works.

Using `user-select: none` on both the window object and the AUT iframe would have done the trick and would have been my first idea.

But using `drag` instead of `mouse` will block selection natively. 
I would have found this more elegant.

All this is cosmetics and I approve the 4 px fix
